doc/nrf/index.rst

```diff
@@ -9,7 +9,7 @@ The SDK contains optimized cellular IoT (LTE-M and NB-IoT), Bluetooth Low Energy
 The |NCS| includes the Zephyr™ real-time operating system (RTOS), which is built for connected low power products.
 
 .. note::
-   nRF53 Series devices (which are pre-production) and Thread, Zigbee, and Bluetooth Mesh protocols are supported for development in v1.3.1 for prototyping and evaluation.
+   nRF53 Series devices (which are pre-production) and Thread, Zigbee, and Bluetooth Mesh protocols are supported for development in v1.3.2 for prototyping and evaluation.
```

Comment:
I think this should have been in the release notes instead, so that all the releases that this applies to has this information. Now it is rolling.

Reply:
We have doc sets for each release, so it is in the documentation for each release.